browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2519,67 +2577,64 @@ index 0aca7c32c4568b43e4b901bdbd72c5f5a0bfcd3c..d1f134304639ae3dd9c862cf1f82b025
  				0FD7C21D23CE41E30096D102 /* PlatformWheelEvent.cpp */,
  				935C476A09AC4D4F00A6AAB4 /* PlatformWheelEvent.h */,
  				BCBB8AB513F1AFB000734DF0 /* PODInterval.h */,
-@@ -31402,6 +31425,7 @@
+@@ -32084,6 +32107,7 @@
  				BCCFBAE70B5152ED0001F1D7 /* DocumentParser.h */,
  				AD6E71AA1668899D00320C13 /* DocumentSharedObjectPool.cpp */,
  				AD6E71AB1668899D00320C13 /* DocumentSharedObjectPool.h */,
 +				F050E17323AD6A800011CE47 /* DocumentTouch.cpp */,
  				6BDB5DC1227BD3B800919770 /* DocumentStorageAccess.cpp */,
  				6BDB5DC0227BD3B800919770 /* DocumentStorageAccess.h */,
  				7CE7FA5B1EF882300060C9D6 /* DocumentTouch.cpp */,
-@@ -32419,6 +32443,7 @@
+@@ -33101,6 +33125,7 @@
  				93C4F6EB1108F9A50099D0DB /* AccessibilityScrollbar.h in Headers */,
  				29489FC712C00F0300D83F0F /* AccessibilityScrollView.h in Headers */,
  				0709FC4E1025DEE30059CDBA /* AccessibilitySlider.h in Headers */,
 +				F050E16D23AD66630011CE47 /* TouchList.h in Headers */,
  				29D7BCFA1444AF7D0070619C /* AccessibilitySpinButton.h in Headers */,
  				69A6CBAD1C6BE42C00B836E9 /* AccessibilitySVGElement.h in Headers */,
  				AAC08CF315F941FD00F1E188 /* AccessibilitySVGRoot.h in Headers */,
-@@ -34517,6 +34542,7 @@
+@@ -35216,6 +35241,7 @@
  				6E4ABCD5138EA0B70071D291 /* JSHTMLUnknownElement.h in Headers */,
  				E44614170CD6826900FADA75 /* JSHTMLVideoElement.h in Headers */,
  				81BE20D311F4BC3200915DFA /* JSIDBCursor.h in Headers */,
 +				F050E17823AD70C50011CE47 /* PlatformTouchPoint.h in Headers */,
  				7C3D8EF01E0B21430023B084 /* JSIDBCursorDirection.h in Headers */,
  				C585A68311D4FB08004C3E4B /* JSIDBDatabase.h in Headers */,
  				C585A69711D4FB13004C3E4B /* JSIDBFactory.h in Headers */,
-@@ -36648,9 +36674,11 @@
- 				B2C3DA3A0D006C1D00EF6F26 /* TextCodec.h in Headers */,
- 				26E98A10130A9FCA008EB7B2 /* TextCodecASCIIFastPath.h in Headers */,
- 				DF95B14A24FDAFD300B1F4D7 /* TextCodecCJK.h in Headers */,
+@@ -37354,6 +37380,8 @@
+ 				A7151BD812F1558F005A0F64 /* TextCheckerClient.h in Headers */,
+ 				A77D0012133B0AEB00D6658C /* TextChecking.h in Headers */,
+ 				A7DBF8DE1276919C006B6008 /* TextCheckingHelper.h in Headers */,
 +				F050E16823AC9C080011CE47 /* PlatformTouchEvent.h in Headers */,
```

Comment:
Loos like those entries that were added in playwright fork not in the alphabetic order. We should fix that, otherwise the diffs like this look odd.